### ISSUE TITLE
oref0-setup can create incompatible config for xdrip-js Logger

### ISSUE THREAD

**Issue by samgladstone** [openaps/oref0#1397]
**Describe the bug**
xdrip-js Logger does not support token-based authentication with Nightscout, however the oref0-setup script will allow you to configure it this way

**To Reproduce**
1. Run oref0-setup
2. Select CGM method: `xdrip-js`
3. Use a NightScout authentication token
4. After set up completes, nothing will be uploaded to nightscout

**Expected behavior**
You should be prompted for an NS API secret rather than API token if you select xdrip-js so that the initial config can work

**Comment by inventor96** [comment]
I believe @sulkaharo knows more, but I believe Nightscout actually automatically handles tokens through the same channels that the API secret is used. I'm currently running the dev branch of oref0 with a NS token on my Edison/Explorer rig plus Logger with my Dexcom G6, and I'm getting live data with no apparent issues into Nightscout. I haven't dug into the logs on the rig to see if there's anything there, but I would imagine there's probably not much if I'm getting Logger info in Nightscout.

**Comment by sulkaharo** [comment]
Yup the newer Nightscout versions should accept tokens using the older API secret methods.
